### ISSUE TITLE
Error: Some of the cache files do not exist 

### ISSUE THREAD

**Issue by shntnu** [broadinstitute/2021_09_01_VarChAMP-data#1]
We get this error. Essentially, none of the files in https://github.com/broadinstitute/2021_09_01_VarChAMP-data/tree/master/profiles/2023_02_23_Batch_3 are available in DVC. 

```
2021_09_01_VarChAMP-data$ dvc pull
WARNING: Some of the cache files do not exist neither locally nor on remote. Missing cache files:                                                                                                                               
name: profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1_augmented.csv.gz, md5: 6aa5a79439ec99e31d936d5314dd5285
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3.csv.gz, md5: 54a45306860161b1bf5e826e0d8c77a2
name: profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1_normalized_feature_select_batch.csv.gz, md5: 9047905062ce70aa999608a290d80da2
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3_normalized_feature_select_batch.csv.gz, md5: 4ff099eb4f5e80af120a38afc7e87575
name: profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1.csv.gz, md5: 8edc9c47f492bd3f1cd22117c2e1c556
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2_normalized.csv.gz, md5: 77f86b8f51d3e39391e5a28151316902
name: profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1_normalized.csv.gz, md5: 53553814fb8c2cfce4b6a491ec91ac06
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2_normalized_feature_select_batch.csv.gz, md5: 3a120dbdff650ac04e07281cfd705110
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2_augmented.csv.gz, md5: 58aa519e29eb0d7eaecb827cdde7dfda
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3_augmented.csv.gz, md5: 543962163ad104203c623fdeba3d91bf
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3_normalized.csv.gz, md5: db2ee77bb01f16b50ff041196fc97314
name: profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2.csv.gz, md5: f4e38dce6114477b9db5e019d3e6766a
12 files failed                                                                                                                                                                                                                 
ERROR: failed to pull data from the cloud - Checkout failed for following targets:
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3_augmented.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1_augmented.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1_normalized_feature_select_batch.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2_normalized_feature_select_batch.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3_normalized_feature_select_batch.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2_augmented.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2_normalized.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P2/2022_09_29_PPL9_P2.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_10_11_PPL9_P1/2022_10_11_PPL9_P1_normalized.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3.csv.gz
/Users/shsingh/work/projects/2021_09_01_VarChAMP/2021_09_01_VarChAMP-data/profiles/2023_02_23_Batch_3/2022_09_29_PPL9_P3/2022_09_29_PPL9_P3_normalized.csv.gz
Is your cache up to date?
<https://error.dvc.org/missing-files>
```

**Comment by bethac07** [comment]
Pushed, apologies! Something else going on here also though when I went digging - Rebecca's files weren't in DVC at all (there are no `.dvc` files and I had to initially set up DVC [later](https://github.com/broadinstitute/2021_09_01_VarChAMP-data/commit/4d16f575a274cf7e4e682cc840fdd69f7b4bbc40) - I suspect hers are being pulled by LFS.

**Comment by shntnu** [comment]
Sorry for the slow turnaround. We are now having issues with AWS permissions. I'll make a note of it here and then ask in Slack. 

```
dvc pull

ERROR: unexpected error - Access Denied: An error occurred (AccessDenied) when calling the ListObjectsV2 operation: Access Denied   
```

```
cat .dvc/config 

[core]
    remote = s3storage
['remote "s3storage"']
    url = s3://imaging-platform-ssf/projects/2021_09_01_VarChAMP_Vidal_Taipale/workspace/software/2021_09_01_VarChAMP_Vidal_Taipale_DVC
```

```
aws s3 ls --no-sign-request s3://imaging-platform-ssf/projects/2021_09_01_VarChAMP_Vidal_Taipale/workspace/software/2021_09_01_VarChAMP_Vidal_Taipale_DVC/

An error occurred (AccessDenied) when calling the ListObjectsV2 operation: Access Denied
```

**Comment by bethac07** [comment]
Do you have access permissions for that bucket? Did you run the profile configuration for accessing that account? See note here https://special-spoon-5b06abaf.pages.github.io/docs/Make_Profiles.html?highlight=dvc

**Comment by shntnu** [comment]
Now fixed in #2
